src/devices/honeywell.c

```diff
@@ -0,0 +1,98 @@
+/*
+* *** Honeywell (Ademco) Door/Window Sensors ***
+*
+* Tested with the Honeywell 5811 Wireless Door/Window transmitters
+*
```

Comment:
If you bother add a tiny description of the bitstream here. You can look at other files for examples.